companies/playmobil-de.json

```
@@ -0,0 +1,21 @@
+{
+    "slug": "playmobil-de",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce",
+        "entertainment"
+    ],
+    "name": "geobra Brandstätter Stiftung & Co. KG",
+    "address": "geobra Brandstätter Stiftung & Co. KG\nBrandstätterstraße 2 - 10\nD - 90513 Zirndorf\nDeutschland",
+    "phone": "+49 911 96660",
+    "fax": "+49 911 96661120",
+    "email": "dsb@playmobil.de",
+    "web": "https://www.playmobil.de",
+    "sources": [
+        "https://www.playmobil.de/startseite"
```

If you middleclick the link, you get the actual URL. :D 

```suggestion
        "https://www.playmobil.de/content/privacy_policy/PRIVACY_POLICY.html"
```